src/StudentsController.ts

```diff
@@ -415,4 +422,48 @@ export class StudentsController {
 
         return { ...student.toJSON(), status: status };
     }
+
+    public async getStudentFiles(student: Student): Promise<SchoolFileDTO[]> {
```

Comment:
I guess the logic returning "files" here differs from the logic within the app showing files of a contact. I like the approach, but we might end up running in problems in future, e.g. if one school allows mails with attachments from the user to the school. It all boils down to our complex file handling...

Reply:
yeah, this massively differs as we know how files are transferred. I think we can definitely improve this in the future.

Reply:
@stnmtz is this a blocker or can we roll out this and improve it later?

Reply:
No, leave it as is, I just wanted to make the comment